app/views/admin/users/show/reports/_index.html.erb

```diff
@@ -28,7 +28,7 @@
   </div>
 <% else %>
   <div class="align-center flex flex-col justify-center my-auto py-8">
-    <h3 class="fs-xl fw-normal mb-2">🧐 No reports submitted.</h3>
-    <p class="color-secondary">This user hasn't reported anything yet.</p>
+    <h3 class="fs-xl fw-normal mb-2">No comment or article has been reported yet.</h3>
+    <p class="color-secondary">Any comment or article reported by <%= @user.name %> will be found here.</p>
```

Comment:
The copy says that this should read the way I have it written, but I am curious: Should we be referring to articles as "articles" or "posts"? I ask because on the non-Admin side of the app, we use the word "Posts" in a lot, if not all, areas, but on the Admin side of the app, we seem to primarily use the word "Articles." 🤔 

cc @Ridhwana

Reply:
Good catch @juliannatetreault.  I'm going to "block" this PR so we don't have a "merge by any core" and let @Ridhwana catch up with the decision.  Please feel free to remove my block once @Ridhwana's weighed in.

Reply:
@juliannatetreault thanks for bringing this up, let's change the wording to "posts" as per [this conversation](https://forem-team.slack.com/archives/CAA200SH5/p1635515334019800) in Slack. I'll also add an issue to our board to rename "Articles" to "Posts" in the admin UI.

Reply:
Thanks, @Ridhwana! 🙌